### ISSUE TITLE
Notes regarding mechanical turret.

### ISSUE THREAD

**Issue by Lgmrszd** [RCXcrafter/Mystical-Gears#13]
I've played around with this new feature and noticed a few things that might have some improvements:
- I don't know much about entities, but turret is considered as "living" one I guess — it gets health bar displayed with Neat (and probably another similar mods) and has health indicator when operated, similar to the one you see when riding a horse.
- Turret base occupies only one block, so the rest space can be covered in blocks, including the space where player sits. This also means many turrets can be placed next to each other, overlapping.

**Comment by Lgmrszd** [comment]
Another thing I just noticed: you can actually kill turrets (with another turret with Ember Gatling Gun installed for example), and they drop sulfur for some reason? And they also respawn instantly.

**Comment by RCXcrafter** [comment]
The turret is considered a living entity, I did this to make the rotation smoother in multiplayer, I didn't think it would cause issues like this so I will make it a more generic entity and see if I can make the movement smooth without being living.
I was able to reproduce the killing issue (which will be fixed by making it not alive) but not the sulfur dropping, that might be caused by another mod messing with drops.
I don't think I will change the hitbox of the turret, I don't want to restrict where people can place blocks around the turret too much.
